### PR TITLE
DIGI_8_GetAllAddOns

### PR DESCRIPTION
**Pull request by bommavikas**
1)Created AddOnDataModel for getAddOns From DataBase
2)Created AddOnController for getAddOns From DataBase

### CODE REVIEW

File: DigiOrderApi/DigiOrderApi/Controllers/AddOnController.cs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+        /// <summary>
+        /// Author: Vikas Bomma
+        /// Date: 7/14/2020
+        /// Description: Get all AddOns from database
+        /// Ref. StoryId: DIGI 8
+        /// Reviewer: 
+        /// Version: 1.0
+        /// </summary>
+        /// <returns></returns>
```
Comment (anebhargavkrishna) [inline DigiOrderApi/DigiOrderApi/Controllers/AddOnController.cs:24]: Remove line 24

File: DigiOrderApi/DigiOrderApi/Controllers/AddOnController.cs [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+            try
+            {
+                addOnDataModel = new AddOnDataModel();
+                dtRecords = addOnDataModel.getAllAddOns();
+                dtRecords.TableName = "ADDONS";
+            }
+            catch (Exception ex)
+            {
+                return Request.CreateErrorResponse(HttpStatusCode.NotFound, "Records Not Found");
+            }
```
Comment (anebhargavkrishna) [inline DigiOrderApi/DigiOrderApi/Controllers/AddOnController.cs:39]: Change to internals ever Error

File: DigiOrderApi/DigiOrderApiDataAccess/AddOnDataModel.cs [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+{
+    public class AddOnDataModel
+    {
+        DataTable dtResult = null;
+
+        //Author: Vikas Bomma
+        //6-25-2020
+        //Description: To get all avaialbel AddOns from database
+        //Reviewer
+        //Review Version:
```
Comment (anebhargavkrishna) [inline DigiOrderApi/DigiOrderApiDataAccess/AddOnDataModel.cs:15]: Remove comments

File: DigiOrderApi/DigiOrderApi/Controllers/AddOnController.cs [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    {
+        AddOnDataModel addOnDataModel = null;
+        DataTable dtRecords = null;
+
+        /// <summary>
+        /// Author: Vikas Bomma
+        /// Date: 7/14/2020
+        /// Description: Get all AddOns from database
+        /// Ref. StoryId: DIGI 8
+        /// Reviewer:
```
Comment (anebhargavkrishna) [inline DigiOrderApi/DigiOrderApi/Controllers/AddOnController.cs:21]: Add Review name Bhargavkrishna Ane

File: DigiOrderApi/DigiOrderApi/Controllers/AddOnController.cs (hunk 2 above)
Comment (bommavikas) [reply DigiOrderApi/DigiOrderApi/Controllers/AddOnController.cs:38]: Changged to Internal Server Error

File: DigiOrderApi/DigiOrderApi/Controllers/AddOnController.cs (hunk 1 above)
Comment (bommavikas) [reply DigiOrderApi/DigiOrderApi/Controllers/AddOnController.cs]: Removed The Line

File: DigiOrderApi/DigiOrderApi/Controllers/AddOnController.cs (hunk 2 above)
Comment (bommavikas) [reply DigiOrderApi/DigiOrderApi/Controllers/AddOnController.cs:38]: Changed To Internal Server Error

File: DigiOrderApi/DigiOrderApiDataAccess/AddOnDataModel.cs (hunk 3 above)
Comment (bommavikas) [reply DigiOrderApi/DigiOrderApiDataAccess/AddOnDataModel.cs]: Removed Comments

File: DigiOrderApi/DigiOrderApi/Controllers/AddOnController.cs (hunk 4 above)
Comment (bommavikas) [reply DigiOrderApi/DigiOrderApi/Controllers/AddOnController.cs]: Added Review Name

**Pull request merged**